chatwoot/chatwoot
# feat: IMAP Email Channel

This change allows the user to configure both IMAP and SMTP for an email inbox. IMAP enables the user to see emails in Chatwoot. And user can use SMTP to reply a email conversation.

User can use the default settings to send and receive emails for email inbox if both IMAP and SMTP are disabled.

Fixes #2520

## Type of change


## How Has This Been Tested?

1. Configure both IMAP and SMTP for outlook email.
2. Send a mail from gmail.
3. Check new emails as conversations in Chatwoot after configuring the IMAP.
4. Reply to the email conversation from Chatwoot.
5. Check for reply mail in gmail.
6. Verify the outlook email ID in the reply mail in gmail.

## Checklist:
